scenes/game/local_game.gd

```diff
@@ -3410,13 +3525,13 @@ func in_range(attacking_player, defending_player, card):
 		var defending_speed = calculate_speed(defending_player, active_strike.get_player_card(defending_player))
 		if attacking_speed > defending_speed:
 			return false
-	if min_range <= distance and distance <= max_range:
-		return true
-	return false
+	return opponent_in_range
 
 func calculate_damage(offense_player : Player, defense_player : Player, offense_card : GameCard, defense_card : GameCard) -> int:
 	var power = get_card_stat(offense_player, offense_card, 'power')
 	var damage = power + offense_player.strike_stat_boosts.power
+	if power == -1:
```

Comment:
Rather than do this, maybe we should just add this to block:
"effect_type": "attack_does_not_hit"
Makes it a lot more explicit than this fragile -1 power thing.

Reply:
If we really want the power to be right, maybe add a "total_power_is_always_X" effect, I think that'll come in handy for Orie too.

Reply:
not hitting should be taken care of by the range fix; i'll try adding the effect just to be safe

Reply:
The main reason I also don't want you to do this here is that I changed this with the Phonon change, so you'll have to merge some conflicts.
I think it's much better to not have -1 be a special power number.
Similarly, if block had the attack_does_not_hit effect, then the range would also be irrelevant.

Reply:
ooh I see; the phonon conflicts should all be merged now and I borrowed the idea of overwriting power, so should be fine on the current version (though I can throw on attack_does_not_hit quickly)